src/main/java/org/candlepin/insights/task/queue/kafka/KafkaTaskProcessor.java

```diff
@@ -57,6 +58,12 @@ public void onTaskAvailable(TaskMessage taskMessage) {
             // not get retried.
             log.error("Failed to execute task: {}", taskMessage, e);
         }
+        finally {
+            // We always ack the message regardless of if there are failures.
+            // There is no need to retry the message on failure since the task
+            // either be manually re-triggered or will run on the next schedule.
```

Comment:
Missing word here.  I'll approve the PR, but if you want to correct the typo you can just accept this suggestion before I merge.

```suggestion
            // can either be manually re-triggered or will run on the next schedule.
```